addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/internal/handler/HeliosHandler221.java

```diff
@@ -1007,7 +1007,7 @@ public void filter(ClientRequestContext requestContext) throws IOException {
         private String getBasicAuthentication() {
             String token = this.user + ":" + this.password;
             try {
-                return "Basic " + DatatypeConverter.printBase64Binary(token.getBytes("UTF-8"));
+                return "Basic " + Base64.getEncoder().encodeToString(token.getBytes("UTF-8"));
```

Comment:
We could use `StandardCharsets.UTF_8` instead of `"UTF-8"` to omit the check for `UnsupportedEncodingException`.

```suggestion
                return "Basic " + Base64.getEncoder().encodeToString(token.getBytes(StandardCharsets.UTF_8));
```